src/OrchardCore.Modules/OrchardCore.Email/Views/SmtpSettings.Edit.cshtml

```diff
@@ -83,10 +83,10 @@
                         <option value="@SmtpEncryptionMethod.None" selected="@(SmtpEncryptionMethod.None == Model.EncryptionMethod ? true : false)">
                             @T["None"] - @T["Connect to server using insecure connection."]
                         </option>
-                        <option value="@SmtpEncryptionMethod.SSLTLS" selected="@(SmtpEncryptionMethod.SSLTLS == Model.EncryptionMethod ? true : false)">
+                        <option value="@SmtpEncryptionMethod.SslTls" selected="@(SmtpEncryptionMethod.SslTls == Model.EncryptionMethod ? true : false)">
                             @T["SSL/TLS"] - @T["Connect to server using SSL/TSL secure connection"]
                         </option>
-                        <option value="@SmtpEncryptionMethod.STARTTLS" selected="@(SmtpEncryptionMethod.STARTTLS == Model.EncryptionMethod ? true : false)">
+                        <option value="@SmtpEncryptionMethod.StartTls" selected="@(SmtpEncryptionMethod.StartTls == Model.EncryptionMethod ? true : false)">
```

Comment:
Compare with case insensitivity for previously set values?

Reply:
Do you mean in `selected` attribute?

Reply:
> Do you mean in `selected` attribute?

Yes, the equality in `selected`.

Reply:
I didn't change it because it was there before ;)

Reply:
I think it's fine to use the equality because we are comparing enum values not strings, right?

Reply:
> I think it's fine to use the equality because we are comparing enum values not strings, right?

Indeed.